conda-forge.yml

```
@@ -1,6 +1,9 @@
+conda_build:
+  pkg_format: '2'
+conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
-conda_forge_output_validation: true
-conda_build:
-  pkg_format: '2'
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```

```suggestion
  linux_ppc64le: azure
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```

Consider cross compiling top speed things up in a separate PR.

---

Done: https://github.com/conda-forge/perl-mime-base64-feedstock/pull/4